app/views/idv/shared/_back.html.erb

```diff
@@ -11,9 +11,8 @@ locals:
 * fallback_path: (Optional) Path to redirect absent action, step, and HTTP referer.
 %>
 <% text = '‹ ' + t('forms.buttons.back')
-   step_url = local_assigns[:step_url] || @step_url
    step = local_assigns[:action] || local_assigns[:step]
-   path = (step_url && step) ? send(step_url, step: step) : go_back_path
```

Comment:
Can we update the code documentation above to remove `action` and `step_url` if they're no longer relevant?